### DISCUSSION TITLE
Textarea in A4 size.

### DISCUSSION THREAD

**Discussion by luqzi99** [tinymce/tinymce#8643] [General]
Hello hi, my project require the textarea of the editor in A4 size. Is it possible to do it on Tinymce version 6 ? if possible, how can i make it. It might look like this. 

![image](https://user-images.githubusercontent.com/55619414/231672874-be55ed7f-52f5-484e-a0a2-b5b7de368a0f.png)
This is from TinyMce v5. https://codepen.io/jvillanueva/pen/oZmxZE

**Comment by lostkeys** [comment]
@luqzi99, you would do the exact same in TinyMCE 6.
